### DISCUSSION TITLE
Update number of processors on forward node

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#4561] [Unsupported Versions]
Are there any tricks to increasing the number of processors the SO software is aware of on a sensor node after install? Say a VM is spun up with X number of processors by mistake, or requirements change, and you allocate more cores to that VM, what are the required files/fields to updated?

**Comment by weslambert** [comment]
Some (located in `/opt/so/saltstack/local/pillar/minions/*`) you might consider are:

`zeek_lbprocs`
`suriprocs`
`ls_pipelineworkers`

**Reply by darth-drew** [reply]
Thanks wes. Do changes here require a reboot?

**Reply by weslambert** [reply]
You may need to reboot for the VM to recognized the extra cores, but from the Security Onion perspective, a restart of the associated services should be enough.

**Reply by darth-drew** [reply]
Sounds good Wes. Thanks!
